addons/purchase_stock/models/stock_move.py

```diff
@@ -159,13 +159,13 @@ def _generate_valuation_lines_data(self, partner_id, qty, debit_value, credit_va
             }
         return rslt
 
-    def _account_entry_move(self, qty, description, svl_id, cost):
+    def _account_entry_move(self, qty, description, svl_id, cost, is_in_out = False):
```

Comment:
Wouldn't something like `move_directions` or `move_types` be easier to understand than `is_in_out` ? I'm expecting a boolean when I'm reading this name with the `False` default value, not really a set.